### DISCUSSION TITLE
CORS Policy issue when intergrating ackee with Next.js

### DISCUSSION THREAD

**Discussion by milindsoorya** [electerious/Ackee#270] [Q&A]
I tried to install ackee using this article [https://kickstart.app/blog/web-analytics-with-ackee-on-vercel](https://kickstart.app/blog/web-analytics-with-ackee-on-vercel). I used the vercel route and after deploying it to production getting CORS error, please help. 
This is the error 
`Access to XMLHttpRequest at 'https://ackee-analytics-orpin.vercel.app/api' from origin 'https://www.picturetalkies.in' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
`

**Comment by electerious** [comment]
It sounds like the `ACKEE_ALLOW_ORIGIN` environment variable is either not set or incorrectly configured.

Here're a few links that could help:
https://github.com/electerious/Ackee/blob/master/docs/CORS%20headers.md#platforms-as-a-service-configuration
https://github.com/electerious/Ackee/blob/master/docs/Options.md#cors-headers
https://github.com/electerious/Ackee/blob/master/docs/Get%20started.md#with-vercel
